sdks/sdk-core/src/chains.ts

```diff
@@ -25,6 +25,7 @@ export enum ChainId {
   ZKSYNC = 324,
   WORLDCHAIN = 480,
   ASTROCHAIN_SEPOLIA = 1301,
```

Comment:
i've been thinking whether we should rename ASTROCHAIN_SEPOLIA to UNICHAIN_SEPOLIA, now we are here. Then make it a breaking change, by titling PR breaking(sdk-core): ...

Reply:
agreed!

Reply:
updated